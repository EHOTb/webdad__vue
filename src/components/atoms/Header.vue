<template>
    <div :class="$style.header">
        <p>to do list</p>
    </div>
</template>

<script>
export default {
    name: "Header",
    props: ['name']
}
</script>

<style lang="scss" module>
@import "../../assets/styles/var.scss";

.header {
    background-color: $bg-color-dark;
    height: 3.1rem;
    display: flex;
    justify-content: center;
    align-items: center;
}
</style>