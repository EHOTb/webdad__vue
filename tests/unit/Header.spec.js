import { mount } from "@vue/test-utils";
import Header from "../../src/components/atoms/Header";

describe("Компонент Header", () => {
  it("Snapshot Checkbox", () => {
    const wrapper = mount(Header);
    expect(wrapper.element).toMatchSnapshot();
  });
});
