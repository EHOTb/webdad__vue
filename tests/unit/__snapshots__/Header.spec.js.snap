// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компонент Header Snapshot Checkbox 1`] = `
<div
  class="header"
>
  to do list
</div>
`;
